File: templates/templates/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/_BenchmarkName_.cs
using System;
using BenchmarkDotNet;
using BenchmarkDotNet.Attributes;

namespace _BenchmarkProjectName_
{
#if(config)
    [Config(typeof(BenchmarkConfig))]
#endif
    public class $(BenchmarkName)
    {
        [Benchmark]
        public void Scenario1()
        {
            // Implement your benchmark here
        }

        [Benchmark]
        public void Scenario2()
        {
            // Implement your benchmark here
        }
    }
}

File: templates/templates/BenchmarkDotNet.BenchmarkProjectTemplate.VB/_BenchmarkName_.vb
Imports System
Imports BenchmarkDotNet
Imports BenchmarkDotNet.Attributes

Namespace _BenchmarkProjectName_

#If config Then
    <Config(GetType(BenchmarkConfig))>
#End If
    Public Class $(BenchmarkName)
        <Benchmark>
        Public Sub Scenario1()
        
        ' Implement your benchmark here

        End Sub

        <Benchmark>
        Public Sub Scenario2()
        
        ' Implement your benchmark here

        End Sub
    End Class
End Namespace